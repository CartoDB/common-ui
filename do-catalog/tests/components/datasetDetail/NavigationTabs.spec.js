import { createLocalVue, shallowMount } from '@vue/test-utils';
import Vuex from 'vuex';
import NavigationTabs from '@/components/datasetDetail/NavigationTabs';
import dataset from '../../fixtures/dataset';

const localVue = createLocalVue();
localVue.use(Vuex);

describe('NavigationTabs.vue', () => {
  const store = createStore();

  it('Renders view properly', () => {
    const navigationTabs = shallowMount(NavigationTabs, {
      localVue,
      store
    });
    expect(navigationTabs).toMatchSnapshot();
  });
});

function createStore(data = dataset.dataset_samples[0]) {
  const store = new Vuex.Store({
    state: {
      catalog: {
        dataset: data
      }
    }
  });
  return store;
}
