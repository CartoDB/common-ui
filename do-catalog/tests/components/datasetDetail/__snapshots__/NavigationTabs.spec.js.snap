// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationTabs.vue Renders view properly 1`] = `<div id="nav" class="nav-tabs u-mb--12"></div>`;
